### ISSUE TITLE
enterprise extend org rules - require pp always in session

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-browser#1820]

**Comment by tomholub** [comment]
should be done after #2011 and #2123

**Comment by tomholub** [comment]
But less important after #1807

**Comment by tomholub** [comment]
Since session become the default in #1807 , this issue is less important. But as a part of this issue in the future, there should be `Rule` flag `FORBID_STORING_PASSPHRASE` that would be affecting setup screen, settings key import screen as well as pass phrase entry screen to forbid storing pass phrase. If enabled, it's only possible to save in session, not in permanent storage.

**Comment by tomholub** [comment]
duplicate of #2752
